src/http_downloader.js

```diff
@@ -52,34 +52,36 @@ class HttpDownloader extends EventEmitter {
 		}
 	}
 
-	downloadResource(resource) {
+	downloadResource(name, resource) {
 		const url = new URL(resource['url']);
-		const name = resource['destination'];
-		const destination = path.join(springPlatform.writePath, name);
+		this.name = name;
+		const destination = path.join(springPlatform.writePath, resource['destination']);
 		if (fs.existsSync(destination)) {
-			this.emit('finished', `Skipping ${destination}: already exists.`);
+			// this.emit('finished', `Skipping ${destination}: already exists.`);
```

Comment:
Plase don't keep old version of the code as comment here and in other locations. Old version is visible in git history and such comment doesn't add any value to reader.

Reply:
Alright, i'll remove it.